pkg/config/piped.go

```diff
@@ -256,6 +273,62 @@ type HelmChartRepository struct {
 	Password string `json:"password"`
 	// Whether to skip TLS certificate checks for the repository or not.
 	Insecure bool `json:"insecure"`
+
+	// Configuration for GIT type.
+	// Remote address of the Git repository used to clone Helm charts.
+	// e.g. git@github.com:org/repo.git
+	GitRemote string `json:"gitRemote"`
+	// The path to the private ssh key file used while cloning Helm charts from above Git repository.
+	SSHKeyFile string `json:"sshKeyFile"`
+}
+
+func (r *HelmChartRepository) IsHTTPRepository() bool {
+	return r.Type == HTTPHelmChartRepository
```

Comment:
```suggestion
	return strings.ToUpper(r.Type) == HTTPHelmChartRepository
```
nits, users may insert HTTP, http or Http as well. Or we may need to check for the type field in the validation function at L293.

Reply:
I don't think that we should allow that. Like, `KubernetesApp` not `K8sApp` or `kubernetesApp`.

Reply:
I also think we should distinguish lowercase and uppercase

Reply:
Sure, in that case, we may need to add validate `r.Type` to the validate function at line L293, and the current error message (at L311) return `Git` instead of `GIT`
https://github.com/pipe-cd/pipe/pull/2863/files#diff-49258a851400c4bc54c40491b440d7edf1dd45eda52c9eccd3f5371af15e8409R311

Reply:
Yep, that should be:
```go
return fmt.Errorf("either %s repository or %s repository must be configured", HTTPHelmChartRepository, GITHelmChartRepository)
```

Reply:
Nice catch!
Fixed it.